src/question_tree.rb

```diff
@@ -45,20 +46,38 @@ def identifiers
       filtered_each.map(&:identifier)
     end
 
+    # START REMAPPING 'content' to variable names
+    delegate :choices, :optional, :type, to: :os_content
```

Comment:
Generally it's clearest to keep all the macros for defining things on a class (`delegate`, `alias` etc) up the top of whatever privacy section they're in (i.e. at the top of the whole class for public methods, or up the top of the `private` section for private methods. Otherwise these may get lost/repeated amongst the rest of the normal method definitions.

Reply:
Done